### PR TITLE
load tos xlsx sheet

### PR DESCRIPTION
**Pull request by MadinaHasan19**
I removed the empty lines and added a return line.

### CODE REVIEW

File: R/tos.R [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#' Load TOS files
+#'
+#' @param xls_file str Path File to Excel file from which TOS are to be loaded.
+#' @param sheet str Name of worksheet to be loaded.
+#' @param sheet_number int Number of worksheet to be loaded
+#' @export
+#'
+load_xls <- function(xls_file, sheet = "HES APC TOS", sheet_number = NA) {
+  myCols <- as.character(read_excel(xls_file, sheet, n_max = 1, skip = 1, col_names = FALSE))
+  print(myCols)
```
Comment (Joe-Heffer-Shef) [inline R/tos.R:10]: remove print statement

**Review by Joe-Heffer-Shef — CHANGES_REQUESTED** [review]
Good work!

A small change is needed.

File: R/tos.R (hunk 1 above)
Comment (MadinaHasan19) [reply R/tos.R]: Thanks I have removed it. Committed the changes and pushed them. 
I am now trying to do a pull request but every time I try to do it shows a compare issue.

Comment (Joe-Heffer-Shef) [reply R/tos.R]: Okay, let's resolve this next time we meet

Comment (ns-rse) [reply R/tos.R]: You don't need to create a new Pull Request @MadinaHasan19.

When there is a Pull Request already been made from a branch any new commits will be added into that pull request. As you've removed the line and pushed it appears below (you can see it on the main [pull request page](https://github.com/CUREd-Plus/cuRed/pull/68) underneath this comment...

image

It has the hash `8c92c83` and you can click on that to see the change.

If you click on the _Files changed_ tab at the top of the pull request and navigate to this line you will see it reflects your change.  This comment is also now marked as _Outdated_ as something has changed in the code (in this instance you've removed the `print()` statement.

Hope that helps

**Comment by MadinaHasan19** [comment]
Thanks a lot, this is so detailed and helpful. I will go through this
tonight and try to improve my understanding of git :)

**Review by Joe-Heffer-Shef — APPROVED** [review]

**Pull request merged**